src/frontend/src/flows/manage/deviceSettings.ts

```diff
@@ -118,8 +128,14 @@ export const deviceSettings = async (
       device,
       isOnlyDevice,
       back: resolve,
-      protectDevice: () =>
-        protectDevice(userNumber, connection, device, isOnlyDevice, resolve),
+      protectDevice: (recoveryDevice: DeviceData & RecoveryDevice) =>
```

Comment:
I find it unfortunate that `protectDevice` now needs to take a device but I get it that it needs a `RecoveryDevice`. We'll be remodelling the device settings page soon so not a big problem.